stock_inventory_revaluation/models/cost_adjustment.py

```diff
@@ -0,0 +1,243 @@
+# Copyright 2021 - Open Source Integrators
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import _, fields, models
+from odoo.exceptions import UserError
+from odoo.tools.float_utils import float_compare
+
+from odoo.addons.base.models.ir_model import MODULE_UNINSTALL_FLAG
+
+
+class CostAdjustment(models.Model):
+    _name = "cost.adjustment"
```

Comment:
IMHO, I would have either split this model in a base module, either you need to rename to something like 'stock.cost.adjustment'.

Reply:
I agree that namespaces are not properly taken case here.
@Chandresh-OSI @patrickrwilson I  hope this has limited impact on the code.